runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -97,4 +97,15 @@ default boolean notifyOnDuel()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			position = 6,
+			keyName = "notifyOnBroadcast",
+			name = "Notify on broadcast",
+			description = "Notifies you whenever Jagex sends a broadcast"
```

Comment:
You need to fix your auto-formatter.
```suggestion
		position = 6,
		keyName = "notifyOnBroadcast",
		name = "Notify on broadcast",
		description = "Notifies you whenever Jagex sends a broadcast"
```